sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Configurations that allow callers to specify details about how to execute
+    /// an Extract Summary action in a set of documents.
+    /// For example, set the model version, specify the order in which extracted
+    /// sentences are returned, and more.
+    /// </summary>
+    public class ExtractSummaryAction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExtractSummaryAction"/>
+        /// class which allows callers to specify details about how to execute
+        /// an Extract Summary action in a set of documents.
+        /// For example, set the model version, specify the order in which extracted
+        /// sentences are returned, and more.
+        /// </summary>
+        public ExtractSummaryAction()
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets a value that, if set, indicates the version of the text
+        /// analytics model that will be used to generate the result.  For supported
+        /// model versions, see operation-specific documentation, for example:
+        /// <see href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/concepts/model-versioning#available-versions"/>.
+        /// </summary>
+        public string ModelVersion { get; set; }
+
+        /// <summary>
+        /// If set, specifies the maximum limit of sentences returned in the result. Defaults to 3.
+        /// </summary>
+        public int? MaxSentenceCount { get; set; }
+
+        /// <summary>
+        /// If set, specifies the order in which the extracted sentences will be returned in the result. Use
+        /// <see cref="SummarySentencesOrder.Offset"/> to keep the original order in which the sentences appear
+        /// in the input. Use <see cref="SummarySentencesOrder.Rank"/> to order them according to their relevance
+        /// to the text input, as decided by the service. Defaults to <see cref="SummarySentencesOrder.Offset"/>.
```

Comment:
```suggestion
        /// to the document input, as decided by the service. Defaults to <see cref="SummarySentencesOrder.Offset"/>.
```

Reply:
Do you think we should do the default in a remarks?

Reply:
We don't do it in the remarks in other properties. For example, in other actions:
```cs
        /// <summary>
        /// The default value of this property is 'false'. This means, Text Analytics service logs your input text for 48 hours,
        /// solely to allow for troubleshooting issues.
        /// Setting this property to true, disables input logging and may limit our ability to investigate issues that occur.
        /// <para>
        /// Please see Cognitive Services Compliance and Privacy notes at <see href="https://aka.ms/cs-compliance"/> for additional details,
        /// and Microsoft Responsible AI principles at <see href="https://www.microsoft.com/ai/responsible-ai"/>.
        /// </para>
        /// </summary>
        /// <remarks>
        /// This property only applies for <see cref="TextAnalyticsClientOptions.ServiceVersion.V3_1"/> and up.
        /// </remarks>
        public bool? DisableServiceLogs { get; set; }
```

So I put it in the summary section for consistency.